tutorials/measurement_example.ipynb

```diff
@@ -59,9 +59,8 @@
    "source": [
```

Comment:
<p>I'd at least cross-link both notebooks. This concrete example should be given as "further reading" in the more general notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BAMWelDX/weldx/pull/326/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I will add the link to the measurement example notebook. I have to rewrite it partially anyways.